tests/ldap/test_extract.py

```diff
@@ -173,3 +177,51 @@ def test_get_merged_ids_by_query_string(
         ldap_persons_with_query, ldap_primary_source
     )
     assert merged_ids_by_query_string == expected
+
+
+def test_get_persons_mocked(ldap_mocker: LDAPMocker) -> None:
+    ldap_mocker([[SAMPLE_PERSON_ATTRS]])
+    persons = get_persons_by_name(surname="Sample", given_name="Sam")
+    persons_list = list(persons)
+    expected = {
+        "company": "RKI",
+        "department": "XY",
+        "departmentNumber": "XY2",
+        "displayName": "Sample, Sam",
+        "employeeID": "1024",
+        "givenName": ["Sam"],
+        "mail": ["SampleS@mail.tld"],
+        "objectGUID": UUID("00000000-0000-4000-8000-000000000000"),
+        "ou": ["XY"],
+        "sAMAccountName": "SampleS",
+        "sn": "Sample",
+    }
+    assert len(persons_list) == 1
+    assert persons_list[0].model_dump(exclude_none=True)["givenName"] == expected["givenName"]
+
+
+def test_get_count_persons_mocked(ldap_mocker: LDAPMocker) -> None:
+    ldap_mocker([[SAMPLE_PERSON_ATTRS]])
+    persons_count = get_count_of_found_persons_by_name(
+        surname="Sample", given_name="Sam"
+    )
+    assert persons_count == 1
+
+
+def test_get_person_mocked(ldap_mocker: LDAPMocker) -> None:
```

Comment:
```suggestion
def test_get_person_by_id_mocked(ldap_mocker: LDAPMocker) -> None:
```

best if the original name of the function you are testing is reflected in the test, so it's easier to find and to indentify